### ISSUE TITLE
Bundler error on install, cannot install ruby_dep

### ISSUE THREAD

**Issue by Jonic** [adrienkohlbecker/vagrant-fsnotify#17]
I'm seeing the following when I try to install vagrant-fsnotify:

```zsh
Bundler, the underlying system Vagrant uses to install plugins,
reported an error. The error is shown below. These errors are usually
caused by misconfigured plugin installations or transient network
issues. The error from Bundler is:

An error occurred while installing ruby_dep (1.4.0), and Bundler cannot continue.
Make sure that `gem install ruby_dep -v '1.4.0'` succeeds before bundling.

Warning: this Gemfile contains multiple primary sources. Using `source`
more than once without a block is a security risk, and may result in installing
unexpected gems. To resolve this warning, use a block to indicate which
gems should come from the secondary source. To upgrade this warning to
an error, run `bundle config disable_multisource true`.
Gem::InstallError: ruby_dep requires Ruby version >= 2.2.5, ~> 2.2.
```

Thing | Version
----- | -------
Vagrant | 1.8.5
Ruby | ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
OS X | 10.11.6

Any ideas? I can't see ruby_dep as a dependency of this plugin, so I'm not sure where that's coming from. Other plugins install fine so I'm not sure if it's something in my version of Vagrant. I'm also a bit puzzled why ruby_dep doesn't see that I'm satisfying its Ruby version requirements! Any help would be very gratefully received!

**Comment by Jonic** [comment]
Turns out that the dependency issue is in the `listen` gem, so I'm closing this.

**Comment by sylar9232** [comment]
Hey @Jonic! How did you fix this issue? Tnx!

**Comment by lothern** [comment]
@sylar9232 I was having this error too. But today I updated Vagrant to 1.8.6 and didn't have an issue installing this plugin. Try updating Vagrant and see if that works for you!

**Comment by Jonic** [comment]
Yep @lothern that did the trick, so I guess it doesn't matter how I solved it in the end!

@sylar9232 Just for the sake of answering the question I forked and cloned this repo, and added a line in the `Gemfile` to read the `.ruby-version` file in my home directory. I built the `.gem` file and told vagrant to install the plugin from there. Because I was telling it to use the correct version of Ruby rather than my system default, it worked and installed successfully, because the ruby_dep gem wasn't complaining about using an incompatible version of Ruby.
